### DISCUSSION TITLE
Change Workspace name, Flow won't recognize collections

### DISCUSSION THREAD

**Discussion by leopascual** [postmanlabs/postman-flows#85] [Defect]
Changed the workspace name from _workspace_ to anything, and **Collections** are not longer accesible from the flow. Even when creating a new Flow can't access endpoints/methods.

**Comment by samay-v** [comment]
Hey @leopascual I am unable to reproduce this defect on the current web app. Could you confirm if this is still happening for you, and if so, which version of postman you're using?

**Comment by leopascual** [comment]
I’m not having the issue anymore.
Thanks

On Wed, 10 Nov 2021 at 13:45 Samay Verma ***@***.***> wrote:

> Hey @leopascual <https://github.com/leopascual> I am unable to reproduce
> this defect on the current web app. Could you confirm if this is still
> happening for you, and if so, which version of postman you're using?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/postmanlabs/postman-flows/discussions/85#discussioncomment-1621983>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKI7VRIV3RBEE7T4WBCLK5DULLDUDANCNFSM5G5S2W5A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 
*--*

1

2

3

4

5

*class contact_info: *

*         name = "Leonardo PASCUAL"*

         phone = +504 99205582

*         follow = ["linkedin
<https://www.linkedin.com/in/leonardo-pascual/>", "facebook
<https://www.facebook.com/jleopascual>", ]*

         skype = "jleopascual"
